content/hi/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Affinity
```

Comment:
I'd expect to see a Hindi term here. Is there any concept in Hindi that would cover affinity in abstract?

In English we can say that magnets have affinity if opposite, and that companies may have affinity if not competitors.

Reply:
Thanks @sftim, I have added the hindi term here as per the context.